tests/docker-compose.yml

```diff
@@ -79,6 +84,16 @@ services:
     depends_on:
       - backend
 
+  modsec3-nginx-debug:
+    <<: *nginx
+    container_name: modsec3-nginx-debug
+    entrypoint: ["/bin/sh", "-c", "/bin/cp /etc/modsecurity.d/owasp-crs/crs-setup.conf.example /etc/modsecurity.d/owasp-crs/crs-setup.conf && /docker-entrypoint.sh nginx-debug -g 'daemon off;'"]
+    environment:
+      <<: *nginx-env
+      LOGLEVEL: "debug"
```

Comment:
Don't set the log level to `debug` when running `nginx-debug`. It's great for looking at how a single requests is processed exactly but it writes a ton of data that will not only slow down everything but might even break stuff (go-ftw, for example, will not be able to find the lines it's looking for because of all the additional stuff in the log).